<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>1159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>設置編輯器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView_Overview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1081</width>
      <height>1091</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1103</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>檔案</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>新增</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>開啟檔案</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>儲存檔案</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
